### PR TITLE
Changing the logs msg for gmail-search-all-mailboxes

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Gmail/Integrations/Gmail/Gmail.yml [hunk 1]
```diff
@@ -645,9 +645,9 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      description: Whether to return only mailboxes which contain the email. Default
+      description: Whether to return the content of the messages as well. Default
         is "False".
-      name: show-only-mailboxes
+      name: return-msg-content
```
Comment (DeanArbel) [inline Packs/Gmail/Integrations/Gmail/Gmail.yml:650]: Please revert this change.
The change should only come from the default value to `true`, and not by changing the arg itself.

File: Packs/Gmail/Integrations/Gmail/README.md [hunk 2]
```diff
@@ -1303,7 +1303,7 @@ Inspecting these messages should allow you to determine what percent the search
 | after | Search for messages sent after a certain time period. For example, 2018/05/06 | Optional | 
 | before | Search for messages sent before a certain time period. For example, 2018/05/09 | Optional | 
 | has-attachments | Whether to search for messages sent with attachments. | Optional | 
-| show-only-mailboxes | Whether to return only mailboxes which contain the email. (Default: false) | Optional | 
+| return-msg-content | Whether to return the content of the messages as well. (Default: false) | Optional |
```
Comment (DeanArbel) [inline Packs/Gmail/Integrations/Gmail/README.md:1306]: also revert this

File: Packs/Gmail/Integrations/Gmail/README.md [hunk 3]
```diff
@@ -1337,7 +1337,7 @@ Inspecting these messages should allow you to determine what percent the search
 
 
 #### Command Example
-```!gmail-search-all-mailboxes after=2019/04/10 max-results=1 before=2021/04/15```
+```!gmail-search-all-mailboxes after=2019/04/10 max-results=1 before=2021/04/15 return-msg-content=true```
```
Comment (DeanArbel) [inline Packs/Gmail/Integrations/Gmail/README.md:1340]: and this

File: Packs/Gmail/ReleaseNotes/1_1_20.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Gmail
+- Note: The ***gmail-search-all-mailboxes*** command will now return by default only the email accounts where the matching messages were found.
```
Comment (DeanArbel) [inline Packs/Gmail/ReleaseNotes/1_1_20.md:4]:
```suggestion
- Note: The ***gmail-search-all-mailboxes*** command will now return by default only the email accounts that have the relevant messages.
```

File: Packs/Gmail/ReleaseNotes/1_1_20.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+- Added the *return-msg-content* argument in the ***gmail-search-all-mailboxes***, if True, the command returns the content of the messages.
+- Deprecated the *show-only-accounts* argument from the ***gmail-search-all-mailboxes*** command.
```
Comment (DeanArbel) [inline Packs/Gmail/ReleaseNotes/1_1_20.md:6]: Please revert

**Review by DeanArbel — CHANGES_REQUESTED** [review]

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 6]
```diff
@@ -1097,9 +1097,12 @@ def get_user_tokens(user_id):
     return result.get('items', [])
 
 
-def search_all_mailboxes(receive_only_accounts):
+def search_all_mailboxes(receive_only_accounts, max_results):
     users_next_page_token = None
     service = get_service('admin', 'directory_v1')
+    accounts_counter = 0
+    msg_counter = 0
+    entry_print_goal = max_results
```
Comment (israelpoli) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:1105]:
```suggestion
    log_msg_target = max_results
```

File: Packs/Gmail/Integrations/Gmail/Gmail.py [hunk 7]
```diff
@@ -1108,26 +1111,33 @@ def search_all_mailboxes(receive_only_accounts):
...
-                            int((accounts_counter / users_count) * 100),
-                            accounts_counter,
-                            users_count,
-                            len(entries)),
-                    )
+                entry, num_of_messages = future.result()
+                entries.append(entry)
+                if receive_only_accounts:
+                    if accounts_counter % 100 == 0:
+                        demisto.info(
```
Comment (israelpoli) [inline Packs/Gmail/Integrations/Gmail/Gmail.py:1128]:
```suggestion
                        demisto.debug(
```
